modules/gitops-argo-cd-dynamic-scaling.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-argo-cd-dynamic-scaling_{context}"]
+= Enabling dynamic scaling of Shards of the Argo CD Application Controller
+
+By default, Argo CD Application Controller assigns clusters to shards indefinitely. This static assignment can result in uneven distribution of shards when replicas are added or removed. You can enable dynamic scaling of shards to automatically adjust the number of shards based on the number of clusters being managed by Argo CD Application Controller at a given time. This ensures that shards are well balanced and provide for the optimal use of compute resources.
+
+.Prerequisites
+* You must have administrator privileges on the cluster.
+* You must have access to the {product-title} web console.
+* You must have installed the {gitops-title} Operator in your cluster.
+
+.Procedure
+
+You can enable dynamic scaling of shards by either using the {product-title} web console or the command-line interface.
+
+. To enable dynamic scaling of shards using the {product-title} web console, follow the following steps:
+.. In the administator perspective of the {product-title} web console, navigate to **Operators** → **Installed Operators**.
+.. From the the list of **Installed Operators**, select the {gitops-title} Operator, and then click on the **ArgoCD** tab.
+.. Select the Argo CD instance name for which you want to enable dynamic scaling of shards, for example "openshift-gitops".
+.. Click on the **YAML** tab, and then edit and configure the `spec.controller.sharding` properties similar to the following example:
```

Comment:
ACK.

I had used 'similar to the following example' in case the user has a custom instance different from the default Argo CD instance installled along side the Operator.

Reply:
@Srivaralakshmi , what do you say?, I need a third opinion here